project-base/storefront/helpers/toasts.tsx

```diff
@@ -1,20 +1,31 @@
 import { isClient } from './isClient';
+import CopyTextBlock from 'components/Basic/CopyTextBlock/CopyTextBlock';
 import { onGtmShowMessageEventHandler } from 'gtm/helpers/eventHandlers';
 import { GtmMessageDetailType, GtmMessageOriginType, GtmMessageType } from 'gtm/types/enums';
+import getConfig from 'next/config';
 import { toast } from 'react-toastify';
 
+const { publicRuntimeConfig } = getConfig();
+
 const showMessage = (message: string, type: 'info' | 'error' | 'success'): void => {
     if (type === 'error') {
-        toast.error(() => <span dangerouslySetInnerHTML={{ __html: message }} data-testid="toast-error" />, {
-            toastId: message,
-        });
-    }
-    if (type === 'info') {
+        if (publicRuntimeConfig.errorMessageVerbosity === 'user') {
```

Comment:
Since we have a lot of places where this `errorMessageVerbosity` check is used, what about add new exported constant like `isWithErrorDebugging`?

Reply:
Done